Add caution about $_POST PHP ini limit while editing large CTs (#754)
ezsystems/developer-documentation: docs/guide/content_model.md

    You can assign each Field defined in a Content Type to a group by selecting one of the groups in the Category drop-down. [Available groups can be configured in the content repository](configuration.md#content-repository-configuration).

!!! caution
    

>     In case of Content Types containing many Field Types you should be aware of possible memory-related issues on publishing/editing. They are caused by the limitation of how many `$_POST` input variables may be accepted. The easiest way to fix them is increasing `max_input_vars` value in `php.ini` configuration file.

Reviewer: ```suggestion
    In case of Content Types containing many Field Types you should be aware of possible memory-related issues on publishing/editing. They are caused by the limitation of how many `$_POST` input variables may be accepted. The easiest way to fix them is increasing `max_input_vars` value in `php.ini` configuration file. You may also experience performance problems with such large Content Types, in particular when you have many Content items.
```
Should we define a threshold, beyond which you should consider doing things differently? 20? 30?

Author: Not sure about this one, haven't tested any number in particular. :/

Reviewer: Yeah, I didn't mean a scientifically valdi limit, just a fencepost-ish thing, saying something like "beyond this you are entering uncharted territory, we don't intend our software to be used like this and haven't tested it, perhaps you should consider other ways of doing what you need to do".
CC @ezsystems/documentation-team

>     In case of Content Types containing many Field Types you should be aware of possible memory-related issues on publishing/editing. They are caused by the limitation of how many `$_POST` input variables may be accepted. The easiest way to fix them is increasing `max_input_vars` value in `php.ini` configuration file.

Reviewer 2: ```suggestion
In case of Content Types containing many Field Types you should be aware of possible memory-related issues with publishing/editing. They are caused by the limitation of how many `$_POST` input variables can be accepted. The easiest way to fix them is by increasing the `max_input_vars` value in the `php.ini` configuration file.
```
